pandas/io/excel.py

```diff
@@ -75,8 +74,19 @@ def read_excel(io, sheetname=0, **kwds):
         The string could be a URL. Valid URL schemes include http, ftp, s3,
         and file. For file URLs, a host is expected. For instance, a local
         file could be file://localhost/path/to/workbook.xlsx
-    sheetname : string or int, default 0
-        Name of Excel sheet or the page number of the sheet
+    sheetname : None, string, int or list of strings and ints, default 0
+        Name, position, or list of names and positions, of Excel sheets.
+        Strings are usd for names, Integers are used in zero-indexed position.
+        When a list is specified, a dictionary of Dataframes are returned,
+        where the keys match the sheet specified.  Set to None to get
+        all sheets, and use sheetnames as the dictionary keys.
+
+        Undefined -> Defaults to 0 -> 1st sheet as a DataFrame
```

Comment:
You will have to use bullet points here to have this formatted nicely in the html docs.